tests/integration/test_integratord/test_integration_options/test_slack_options.py

```diff
@@ -0,0 +1,249 @@
+'''
```

Comment:
Apply the suggested changes to "tests/integration/test_integratord/test_integration_options/test_pagerduty_options.py" , because it has the same comments to apply here.

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/4108/commits/5781f6525ec9626b8aa282b7c7c3738b29bbbf9e